### PR TITLE
Coming Soon Pages Added

### PR DESCRIPTION
**Pull request by ATHARVA-GAWAS**
Issue Solved
#1808 

image

### CODE REVIEW

File: website/templates/bacon.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
```
Comment (arkid15r) [inline website/templates/bacon.html:1]: These pages looks identical. If that's the case I recommend following the DRY principle and crate a "coming soon" template page to use it as a base for the new pages.

Comment (ATHARVA-GAWAS) [reply website/templates/bacon.html:1]: Ok, let me check

**Comment by ATHARVA-GAWAS** [comment]
Done, made a file comingsoon.html and extended it to other pages

**Review by arkid15r — COMMENTED** [review]
I think the naming convention suggests "coming_soon.html".

Other than that looks good 👍 (I didn't review the page's from UI perspective)


upd: correct the extension.

**Comment by AtmegaBuzz** [comment]
ArcoLinux_2024-02-22_00-05-17

For UI i would suggest making it something like this

**Comment by ATHARVA-GAWAS** [comment]
Better?

image

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Can you please make it look exactly like the suggestion?

**Comment by ATHARVA-GAWAS** [comment]
Ok, i will do it

**Comment by ATHARVA-GAWAS** [comment]
Is this fine?

image

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
One tweak

**Comment by ATHARVA-GAWAS** [comment]
Ok

**Comment by ATHARVA-GAWAS** [comment]
Actually base.html is essential, since without it, the entire structure of the page gets messed up

**Comment by ATHARVA-GAWAS** [comment]
Changes done

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**